src/main/java/mate/jdbc/model/Driver.java

```diff
@@ -0,0 +1,70 @@
+package mate.jdbc.model;
+
+import java.util.Objects;
+
+public class Driver {
+    private Long id;
+    private String name;
+    private String licenseNumber;
+
+    public Driver() {
+    }
+
+    public Driver(Long id, String name, String licenseNumber) {
```

Comment:
How should I do that? Сause, in DriverDaoImpl, I created method getDriver which return new Driver(id, name, licenseNumber); So, I don't understand, how I could remove id from constructor. If I remove it, it will be null id.